scalafmt-core/shared/src/main/scala/org/scalafmt/config/Newlines.scala

```diff
@@ -95,8 +95,15 @@ import metaconfig.generic.Surface
   *         ...
   *       }
   *   }}}
+  * @param source
+  *   Controls how line breaks in the input source are handled
+  *   If `classic` (default), the old mixed behaviour will be used
```

Comment:
Users don't know what is "old behaviour" 😄 
In the next steps we must remember to rewrite this more detailed